### ISSUE TITLE
IE11 Error: Could not complete the operation due to error 800a025e (removeAllRangeswindow.getSelection().removeAllRanges())

### ISSUE THREAD

**Issue by mwojtul** [handsontable/handsontable#4620]
### Description
When clicking between cells in a spreadsheet, eventually IE11 throws an error: Could not complete the operation due to error 800a025e.

### Steps to reproduce
1. On IE11 (most likely 9/10 as well, haven't tested), single click on a cell
2. Keep double clicking on different cells until the error is thrown

### Demo
https://forum.handsontable.com/t/javascript-runtime-error-in-internet-explorer/476 (http://jsfiddle.net/aLyyc4ud/)

Note that this issue is fully documented at the link above with detailed repro steps. Seems a fix just never made it into the codebase. 

### Your environment
* Handsontable version: 0.34.5
* Browser Name and version: IE11
* Operating System: Windows 7

**Comment by AMBudnik** [comment]
Thanks for adding @mwojtul 

I have added the connection from https://forum.handsontable.com/t/gh-4620-javascript-runtime-error-in-internet-explorer/476/3 to this issue.

**Comment by AMBudnik** [comment]
I have just tested the following case

**settings**
```js
colHeaders: true,
startRows: 5,
startCols: 5,
type: 'numeric',
```

on my Windows 10, Handsontable 8.0.0-beta2-rev15, and Edge browser
<img width="153" alt="edge_ver" src="https://user-images.githubusercontent.com/10757813/85015512-60626e00-b168-11ea-8223-2977cfc2596b.png">

and **the issues are not replicable.**

As [support for the Internet Explorer browser is to be deprecated in version 8.0.0](https://github.com/handsontable/handsontable/issues/7026) we decided to close all the issues that are only replicable on that browser. All IE-related workarounds implemented into Handsontable are still a part of the code, and we do not plan to remove them.

Thank you for sharing. I will also leave a note on the forum.
